swe_bench/inference/run_api.py

```diff
@@ -0,0 +1,112 @@
+import json
+from pathlib import Path
+
+import fire
+from data.load_dataset import load_oracle_dataset
```

Comment:
load_dataset does not exist in data folder

Reply:
https://github.com/geekan/MetaGPT/pull/1077